charts/quickwit/values.yaml

```diff
@@ -229,20 +256,26 @@ metastore:
 control_plane:
   # Extra env for control plane
   extraEnv: {}
-    # KEY: VALUE
+  # KEY: VALUE
```

Comment:
This comment shouldn't be changed as well

Reply:
Fixed